test/verify/check-services

```diff
@@ -836,6 +836,68 @@ WantedBy=default.target
         b.click(".modal-dialog .btn-danger")
         self.check_service_details(["Masked"], ["Allow running (unmask)"], False, onoff=False)
 
+    @skipImage("Implemented in #12585", "rhel-8-0-distropkg")
```

Comment:
rhel-8-0-distropkg is on its way out (PR #12735). To avoid conflict with that, can you please add rhel-8-1-distropkg to the list? (The two PRs will still conflict either way, though)

Reply:
Done.